### PR TITLE
ShooterPostWaco

### PR DESCRIPTION
**Pull request by aetherstorm-dev**
Some post-Waco shooter updates. Mostly for convenience.

### CODE REVIEW

File: src/main/java/frc/robot/Constants.java [hunk 1]
```diff
@@ -58,10 +58,10 @@ public static class ShooterConstants {
     public static final double SHOOTER_LEFT_D = 0;
     public static final double SHOOTER_LEFT_FF = 0;
 
-    public static final double SHOOTER_P = 0.001;
-    public static final double SHOOTER_I = 0.00000035;
-    public static final double SHOOTER_D = 0;
-    public static final double SHOOTER_FF = 0;
+    public static final double SHOOTER_RIGHT_P = 0.001;
```
Comment (caseyjbrotherton) [inline src/main/java/frc/robot/Constants.java:61]: You will want to merge from main.  There is a new HowdyPID object that is defined for shooter's pid

File: src/test/java/TestShooterSubsystem.java [hunk 2]
```diff
@@ -1,43 +1,18 @@
...
-      speedsPair = ShooterSubsystem.calculateShooterSpeeds(speakerConfigListTest[i][0]);
-      speedsArray[0] = speedsPair.getSpeedLeftInRPM();
-      speedsArray[1] = speedsPair.getSpeedRightInRPM();
-      configSpeeds = arraySlice(speakerConfigListTest[i], 1, 2);
-      Assertions.assertArrayEquals(configSpeeds, speedsArray);
+    for (int i = 0; i < ShooterSubsystem.speakerConfigList.length; i++) {
+      testConfig =
+          ShooterSubsystem.calculateShooterSpeeds(
+              ShooterSubsystem.speakerConfigList[i].getDistanceInInches());
+      Assertions.assertEquals(testConfig, ShooterSubsystem.speakerConfigList[i]);
```
Comment (caseyjbrotherton) [inline src/test/java/TestShooterSubsystem.java:15]: This test only tests exact matches for Shooter Configurations.

We should also consider test cases:

- Is config list in sorted order, with both distance, left speed and right speed increasing as we move through the list?
- Does a distance value below minimum provide the lowest possible speeds?
- Does a distance value above maximum provide the highest possible speed?

**Review by caseyjbrotherton — CHANGES_REQUESTED** [review]
Code looks good.  Much cleaner with equals.

Since we are changing, let's consider adding more test cases to verify operation

File: src/main/java/frc/robot/Constants.java (hunk 1 above)
Comment (aetherstorm-dev) [reply src/main/java/frc/robot/Constants.java]: Resolved in bb39de9

File: src/test/java/TestShooterSubsystem.java (hunk 2 above)
Comment (aetherstorm-dev) [reply src/test/java/TestShooterSubsystem.java:15]: Tests added in 3cc5cbb

**Pull request closed without merging**